tasks/quality_gates.py

```diff
@@ -372,21 +375,24 @@ def debug_specific_quality_gate(ctx, gate_name):
 
 
 @task()
-def exception_threshold_bump(ctx):
+def exception_threshold_bump(ctx, pipelineId):
```

Comment:
```suggestion
def exception_threshold_bump(ctx, pipeline_id):
```
as per PEP-8, whose digest might be found at: https://realpython.com/python-pep8/#naming-styles